src/controllers/settings.controller.js

```diff
@@ -0,0 +1,33 @@
+import ResponseService from '../services/response.service';
+import UserService from '../services/user.service';
+/**
+ *
+ *
+ * @class SettingsController
+ */
+class SettingsController {
+  /**
+  * super_admin can assign a role to user by email
+  * @static
+  * @description PATCH /api/admin/reset-user-role
+  * @param {object} req request object
+  * @param {object} res response object
+  * @memberof UserRoleSettingsController
+  * @returns {object} ResponseService
+  */
+  static async changeUserRole(req, res) {
+    const { userEmail, userRole } = req.body;
+    const findUser = await UserService.findUserByProperty({ email: userEmail.trim() });
+    if (findUser && findUser.role !== 'super_admin') {
+      await UserService.updateUser({ email: userEmail.trim() }, { role: userRole.trim() });
+      const data = await UserService.findUserByProperty({ email: userEmail.trim() });
```

Comment:
You don't have to do another find here. You can return the `findUser` on line 20 and override the role property with the new userRole that was updated.

Reply:
finished now